selectors-4/Overview.bs

```diff
@@ -2256,6 +2256,10 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 		like clicking on the button with a mouse pointer.
 	</div>
 
+	This pseudo-class is intended to give developers the ability to style focus
+	selectively, in order to encourage visible, usable focus styles, and
+	discourage the practice of disabling the default user agent focus style.
+
 	<div class=advisement>
```

Comment:
Should this section be removed, as it's essentially duplicated below?

Reply:
fixed